cnc_ctrl_v1/CNC_Functions.h

```diff
@@ -958,8 +963,7 @@ void  calibrateChainLengths(){
     
     //measure out the left chain
     Serial.println(F("Measuring out left chain"));
-    leftAxis.setPIDAggressiveness(.1);
-    singleAxisMove(&leftAxis, ORIGINCHAINLEN, 500);
+    singleAxisMove(&leftAxis, ORIGINCHAINLEN, 800);
```

Comment:
Should we worry about the chain wrapping more at the faster feedrate of 800?

Reply:
I tried both the chain calibration and machine measurement last night.  It doesn't seem unsafe, but maybe I have been working with it for so long.

Reply:
I tested it too and it seemed very reasonable and the slight speed improvement is very welcome. I think I was just very conservative originally